lambdas/credentialissuerreturn/src/main/java/uk/gov/di/ipv/core/credentialissuerreturn/CredentialIssuerReturnHandler.java

```diff
@@ -64,6 +69,7 @@ public APIGatewayProxyResponseEvent handleRequest(
                 RequestHelper.convertRequest(input, CredentialIssuerRequestDto.class);
 
         var errorResponse = validate(request);
+
         if (errorResponse.isPresent()) {
             return ApiGatewayResponseGenerator.proxyJsonResponse(400, errorResponse.get());
```

Comment:
Could we add an error log in here? That way we can see a bit clearer if this or any of the other errors are thrown

Reply:
Good idea, done!